private.tf

```diff
@@ -63,6 +78,15 @@ resource "aws_network_acl" "private" {
   vpc_id     = var.vpc_id
   subnet_ids = aws_subnet.private.*.id
 
+  ingress {
```

Comment:
if assign_ipv6_address_on_creation is false then these ipv6 ingress and egress shouldn't be set. these can be disabled using a dynamic